GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.Designer.cs

```diff
@@ -418,18 +423,52 @@ private void InitializeComponent()
             this._cloneSetupGB.Controls.Add(this._browseForCloneToDirbtn);
             this._cloneSetupGB.Controls.Add(this._destinationTB);
             this._cloneSetupGB.Dock = System.Windows.Forms.DockStyle.Fill;
-            this._cloneSetupGB.Location = new System.Drawing.Point(3, 369);
+            this._cloneSetupGB.Location = new System.Drawing.Point(3, 351);
             this._cloneSetupGB.Name = "_cloneSetupGB";
-            this._cloneSetupGB.Size = new System.Drawing.Size(738, 140);
+            this._cloneSetupGB.Size = new System.Drawing.Size(738, 158);
             this._cloneSetupGB.TabIndex = 23;
             this._cloneSetupGB.TabStop = false;
             this._cloneSetupGB.Text = "Clone";
             // 
+            // _cloneProtocolGroupBox
+            // 
+            this._cloneProtocolGroupBox.Controls.Add(this._useHttpsRadioButton);
+            this._cloneProtocolGroupBox.Controls.Add(this._useSshRadioButton);
+            this._cloneProtocolGroupBox.Location = new System.Drawing.Point(10, 97);
+            this._cloneProtocolGroupBox.Name = "_cloneProtocolGroupBox";
+            this._cloneProtocolGroupBox.Size = new System.Drawing.Size(200, 18);
+            this._cloneProtocolGroupBox.TabIndex = 25;
```

Comment:
Could you please confirm the tab order is/remains logical. If not, please correct it.

Reply:
Appears to be logical, but I have to admit I hadn't even thought to check
![forkandclonetaborder](https://user-images.githubusercontent.com/16868093/49907052-278e5e00-fe3a-11e8-9f9d-df5e9e3cd96d.gif)

Reply:
👍